test/modules/test_modules_remind.py

```diff
@@ -8,17 +8,18 @@
 import pytest
 import pytz
 
-from sopel import test_tools
 from sopel.modules import remind
 
 
-@pytest.fixture
-def sopel():
-    bot = test_tools.MockSopel('Sopel')
-    bot.config.basename = 'default'
-    bot.config.core.owner = 'Admin'
-    bot.config.core.host = 'chat.freenode.net'
-    return bot
+TMP_CONFIG = """
+[core]
+owner = Admin
+nick = Sopel
+enable =
+    coretasks
```

Comment:
"Explicit is better than implicit", or would `coretasks` not be loaded in the test without this?

Reply:
It's a way to prevent anything else from being loaded by mistake. It's not necessary right now, since `bot.setup` isn't called - but what if it is? But yes, this is not strictly necessary.